shoplist/src/main/java/com/netguru/codereview/ui/model/ShopList.kt

```diff
@@ -0,0 +1,11 @@
+package com.netguru.codereview.ui.model
+
+import com.netguru.codereview.network.model.ShopListItemResponse
+
+class ShopList(
+    val id: String,
+    val userId: Int,
+    val listName: String,
+    val iconUrl: String,
+    val items: List<ShopListItemResponse>
```

Comment:
UI models should be separated from network models. It would be good to create ShopListItem class in package `com.netguru.codereview.ui.model` that will represent `ShopListItemResponse` in UI models
```suggestion
    val items: List<ShopListItem>
```